extension/my-style.js

```diff
@@ -128,7 +140,14 @@
 
     /* Save styles persistently in local storage. */
     var saveStyles = throttle(function() {
-      localStorage.myStyle = style.innerHTML;
+      var obj = {};
+      obj[storageKey()] = style.innerHTML;
+
+      textarea.setAttribute('syncing', true);
+
+      chrome.storage.sync.set(obj, function() {
+        textarea.setAttribute('syncing', false);
```

Comment:
@karthikv I included this in case you wanted to style the textarea a certain way if ever the syncing didn’t complete or took a while. For example:

```css
#my-style-input {
  box-shadow: inset 0 0 red !important;
  transition: box-shadow .5s 2s ease !important;
}

#my-style-input[syncing="true"] {
  box-shadow: inset -5px 0 red !important;
}
```